File: ImageAPI/GetImage.py
import requests

imageId = "ADD IMAGE ID HERE"
x = 31992
y = 20452
z = 16
requestId = "ADD REQUEST ID (returned from Catalog API) HERE"
token = "ADD YOUR TOKEN HERE"

url = ("https://image-api.hibirdi.com/image/id/{}/x/{}/y/{}/z/{}?requestId={}&token={}"
       .format(imageId, x, y, z, requestId, token))

response = requests.request("GET", url)

print(response.text)

File: ImageAPI/GetImage.js
let data = null;
const xhr = new XMLHttpRequest();

const imageId = "ADD IMAGE ID HERE";
const x = 31992;
const y = 20452;
const z = 16;
const requestId = "ADD REQUEST ID (returned from Catalog API) HERE";
const token = "ADD YOUR TOKEN HERE";

const url = "https://image-api.hibirdi.com/image/id/" + imageId +
            "/x/" + x +
            "/y/" + y +
            "/z/" + z +
            "?requestId=" + requestId +
            "&token=" + token;

xhr.addEventListener("readystatechange", function () {
  if (this.readyState === 4) {
    console.log(this.status);
  }
});

xhr.open("GET", url);
xhr.send(data);
